examples/conv-opt/comparison/boost_gil_conv2d.cpp

```diff
@@ -0,0 +1,43 @@
+#include <boost/gil/extension/io/png.hpp>
```

Comment:
Please format the file with `clang-format`.

Reply:
Since I couldn't find a custom styled `.clang-format` file, I have used the default one generated using `clang-format -style=llvm -dump-config > .clang-format`.